lib/views/repository_page.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:github_search_flutter_client_rxdart_example/app/github_search_delegate_repository.dart';
+import 'package:github_search_flutter_client_rxdart_example/models/github_repository.dart';
+import 'package:github_search_flutter_client_rxdart_example/services/github_search_api_wrapper.dart';
+import 'package:github_search_flutter_client_rxdart_example/services/github_search_service_repository.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class RepositoryPage extends StatefulWidget {
+  @override
+  _RepositoryPageState createState() => _RepositoryPageState();
+}
+
+class _RepositoryPageState extends State<RepositoryPage> {
+
+  GitHubRepository repository;
+
+  final estiloTitulo = TextStyle(fontSize: 20.0, fontWeight: FontWeight.bold);
+  final estiloSubTitulo = TextStyle(fontSize: 15.0, color: Colors.grey);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('GitHub Search Repository'),
+        actions: [
+          IconButton(icon: Icon(Icons.search), onPressed: () => _showSearch(context)),
+        ],
+      ),
+      body: (repository == null) 
+      ? Center(
+        child: Text('Repository no selected'),
+      )
+      : SingleChildScrollView(
+        physics: BouncingScrollPhysics(),
+        child: Column(
+          children: <Widget>[
+            _crearImagen(),
+            SizedBox(height: 25.0),
+            _crearTitulo(),
+            SizedBox(height: 25.0),
+            _crearBoton(),
+            SizedBox(height: 25.0),
+            _crearTexto(repository.description),
+            SizedBox(height: 110.0),
+          ],
+        ),
+      )
+    );
+  }
+
+  Widget _crearImagen(){
```

Comment:
https://iiro.dev/2018/12/11/splitting-widgets-to-methods-performance-antipattern/